src/dymaptic.GeoBlazor.Core/Objects/AttributesDictionary.cs

```diff
@@ -16,21 +17,28 @@ namespace dymaptic.GeoBlazor.Core.Objects;
 public class AttributesDictionary : IEquatable<AttributesDictionary>
 {
     /// <summary>
-    ///     Constructor
+    ///     Constructor for a new, empty dictionary
     /// </summary>
     public AttributesDictionary()
     {
         _backingDictionary = new Dictionary<string, object?>();
     }
 
     /// <summary>
-    ///     Constructor from existing dictionary
+    ///     Constructor from existing dictionary or JSON deserialized dictionary
     /// </summary>
     /// <param name="dictionary">
-    ///     The dictionary to use
+    ///     The dictionary to use.
     /// </param>
+    /// <remarks>
+    ///     It is possible to control the culture used for parsing numbers and dates by including a "geoBlazorCulture" key in the dictionary. The value should be a string representation of a culture name, such as "en-US" or "fr-FR".
+    ///     Alternatively, for a server-generated dictionary, the culture can be set in the server's CultureInfo.DefaultThreadCurrentUICulture property. It will fall back to `CultureInfo.CurrentCulture` if not set.
+    /// </remarks>
     public AttributesDictionary(Dictionary<string, object?> dictionary)
     {
+        CultureInfo cultureInfo = dictionary.TryGetValue("geoBlazorCulture", out object? gbCulture)
```

Comment:
Would it make sense to make this a property on the class rather then another entry in the dictionary? It could help separate data vs framework, unless these attributes are not directly realted to the data.

Reply:
The problem is the serialization of the attributes, where we have to switch them to an array of objects with `key`, `value`, and `valueType`. I could rewrite all of that to be an object _with_ an array, but this seemed simpler. Rewriting would mean rewriting both JSON and Protobuf serialization methods in both C# and TypeScript. 😩